### ISSUE TITLE
e_advanced_recon.ipynb

### ISSUE THREAD

**Issue by paskino** [SyneRBI/SIRF-Exercises#88]
data path in `e_advanced_recon.ipynb` is not correct. Requires:

```python
from exercises_data_path import exercises_data_path
#%% GO TO MR FOLDER
os.chdir(os.path.join(exercises_data_path,'MR', 'PTB_ACRPhantom_GRAPPA'))
```

Also this block is wrong because `bwd_img_arr` is 3D.

```python
# PLOT THE RESULTS
grappa_img_arr = norm_array(grappa_images.as_array())
bwd_img_arr = norm_array(bwd_img.as_array())

fig = plt.figure(figsize=(9, 4))
plt.set_cmap('gray')

ax = fig.add_subplot(1,2,1)
ax.imshow(abs(grappa_img_arr), vmin=0, vmax=1)
ax.set_title('Result of GRAPPA Reconstruction')
ax.axis('off')

ax = fig.add_subplot(1,2,2)
ax.imshow(abs(bwd_img_arr), vmin=0, vmax=1)
ax.set_title('Result of AcquisitionModel.backward()')
ax.axis('off')

plt.tight_layout()

>> TypeError: Invalid shape (4, 256, 256) for image data
```

**Comment by KrisThielemans** [comment]
The latter error is because the notebook uses the csm to initialise the acquisition model, but that one has 4 "channels". This was briefly discussed in the tcon on Thursday.

I'll revert this to using the grappa image for now. @johannesmayer can fix it better for 3.1 😄

**Comment by KrisThielemans** [comment]
I've also fixed some grammar mistakes (I always thought that only English people wrote "it's" when they meant "its" 😆 ).

But it's a great notebook! Thanks a lot @johannesmayer
